website/content/docs/upgrading/instructions/upgrade-to-1-10-x.mdx

```diff
@@ -0,0 +1,222 @@
+  Consul Enterprise 1.8.13 and 1.9.7 have the ability to load the license
+  from these configurations, it is intended to only be used during upgrades.
+  Licenses loaded this way are not online reloadable. It is therefore important
+  to promptly proceed with the upgrade and not leave agents in the intermediate state.
+
+**6.** Upgrade all client agents to the latest 1.8.x or 1.9.x release.
+
+**7.** Pre-configure the snapshot agents with a license. This is the same process that
+  was executed for the servers in step 5.
+
+**8.** Upgrade all [snapshot agents](/docs/commands/snapshot/agent) to the latest 1.8.x or 1.9.x release.
+
+**9.** Take a snapshot of the upgraded cluster by running:
+
+```shell
+consul snapshot save intermediate.snap
+```
+
+  This snapshot should not be needed but if something were to go wrong, having
+  the snapshot would make it possible to restore the previous state.
+
+**8.** Pre-configure the server agents with a license. This is the same process that
+  was executed for the servers in step 5.
+
+**9.** Upgrade all server agents to v1.10.0.
+
+**10.** Upgrade all client agents to v1.10.0.
+
+**11.** Upgrade all [snapshot agents](/docs/commands/snapshot/agent) to v1.10.0.
+
+</Tab>
+<Tab heading="Kubernetes">
+
+**4.** If the Consul servers are running outside the Kubernetes cluster, ensure they are upgraded
+to the latest 1.8.x or 1.9.x before the next steps.
+
+**5.** Upgrade to the latest Consul helm chart v0.32.0+.
+
+**6.** Update the value of `global.image` in the values file to the latest 1.8.x or 1.9.x image.
+
+  Additionally, ensure that the Kuberenetes secret with the license is correctly specified in the
+  values `server.enterpriseLicense.secretName` and `server.enterpriseLicense.secretKey`.
+
+**7.** Upgrade the cluster.
+
+  Follow the steps in the [Kubernetes Upgrade Guide](/docs/k8s/upgrade#upgrade-consul-on-kubernetes)
+  to then upgrade the cluster. Ensure you check the steps to upgrade the [servers](/docs/k8s/upgrade#upgrading-consul-servers)
+  and the [clients](/docs/k8s/upgrade#upgrading-consul-clients).
+
+**8.** If the Consul servers are running outside the Kubernetes cluster, ensure they are upgraded
+  to 1.10.0 before the next steps. 
+  
+  This will require pre-configuring the servers with a license
+  before running 1.10. This can either be set with the `license_path` configuration item, the 
+  `CONSUL_LICENSE_PATH` environment variable or the `CONSUL_LICENSE` 
+  environment variable. See the [licensing documentation](/docs/enterprise/license/overview) 
+  for more information about how to configure the license. Note that while
+  Consul Enterprise 1.8.13 and 1.9.7 have the ability to load the license
+  from these configurations, it is intended to only be used during upgrades.
+  Licenses loaded this way are not online reloadable. It is therefore important
+  to promptly proceed with the upgrade and not leave agents in the intermediate state.
```

Comment:
Should this block be here in the Kube guide @thisisnotashwin? It seems odd talking about ENV vars when the use only needs to set things in helm values?

Reply:
never mind this is for external servers case